pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -91,10 +91,16 @@ def prepare() {
         git url: env.OSA_OPS_REPO, branch: "master"
         sh "git checkout ${env.OSA_OPS_BRANCH}"
       }
-      dir("openstack-ansible-ops/${env.MULTI_NODE_AIO_DIR}") {
+      dir("openstack-ansible-ops/multi-node-aio") {
+        // The multi-node-aio tool is quite modest when it comes to allocating
+        // RAM to VMs -- since we have RAM to spare we double that assigned to
+        // infra nodes.
+        sh """#!/bin/bash
+          sed -i 's/server_vm_ram: 8192/server_vm_ram: 16384/g' playbooks/host_vars/infra*.yml
+          cp -a ${WORKSPACE}/rpc-gating/scripts/dynamic_inventory.py playbooks/inventory
+          cp -a ${WORKSPACE}/rpc-gating/playbooks/templates/openstack_user_config.yml.j2 playbooks/osa/openstack_user_config.yml
```

Comment:
I think with https://review.openstack.org/#/c/507886/ we could probably remove this

Reply:
This is a great change, thanks @cloudnull.  Unfortunately we'll still be in the position where either rpc-gating or rpc-openstack needs to carry this because mitaka, liberty, kilo, etc. need to reference os-infra_hosts which isn't in this openstack_user_cnofig.yml file.  Do you have any suggestions to avoid us having to carry that?